Fix command to run migrations

### User Story

As a develop/devops I want to be able to run migrations with `lein run migrate`, as it is specified here: http://www.luminusweb.net/docs/migrations.html

### Description

This is important to separate running the app from running the migrations, that right now it is coupled, and everything the app is run, the migrations are run: https://github.com/status-im/open-bounty/blob/develop/src/clj/commiteth/db/core.clj#L29

Right now that command is broken with this output:

```
Exception in thread "main" java.lang.NullPointerException, compiling:(/private/var/folders/94/ynczyqvs09v8_7t8m21t5tt40000gn/T/form-init1764225723835109313.clj:1:125)
	at clojure.lang.Compiler.load(Compiler.java:7526)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: java.lang.NullPointerException
	at migratus.database$connect_STAR_.invokeStatic(database.clj:120)
	at migratus.database$connect_STAR_.invoke(database.clj:114)
	at migratus.database.Database.connect(database.clj:241)
	at migratus.core$run.invokeStatic(core.clj:24)
	at migratus.core$run.invoke(core.clj:21)
	at migratus.core$migrate.invokeStatic(core.clj:69)
	at migratus.core$migrate.invoke(core.clj:66)
	at luminus_migrations.core$migrate.invokeStatic(core.clj:73)
	at luminus_migrations.core$migrate.invoke(core.clj:52)
	at commiteth.core$_main.invokeStatic(core.clj:60)
	at commiteth.core$_main.doInvoke(core.clj:53)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.lang.Var.invoke(Var.java:381)
	at user$eval30723.invokeStatic(form-init1764225723835109313.clj:1)
	at user$eval30723.invoke(form-init1764225723835109313.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	... 12 more
```

--- a/src/clj/commiteth/core.clj
+++ b/src/clj/commiteth/core.clj
@@ -54,10 +54,9 @@ repl-server
   (cond
     (some #{"migrate" "rollback"} args)
     (do
-      (mount/start
-        #'commiteth.config/env
-        #'commiteth.db.core/*db*)
-      (migrations/migrate args (select-keys env [:jdbc-database-url]))
+      (mount/start #'commiteth.config/env)
+      (migrations/migrate args {:database-url (:jdbc-database-url env)})
+      (log/info "Successfully ran" (first args))
       (System/exit 0))
     :else
     (start-app args)))

